ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/TestData.kt

```diff
@@ -283,7 +284,7 @@ private object Constants {
     val assetBaseURL = URL("https://assets.pawwalls.com")
 }
 
-private data class TestStoreProduct(
+internal data class TestStoreProduct(
```

Comment:
We might want to move this to the main SDK for everyone to use? It's been super helpful in iOS.